pyro/contrib/gp/kernels/kernel.py

```diff
@@ -10,6 +13,16 @@ class Kernel(Parameterized):
     Every inherited class should implement the forward pass which
     take inputs X, Z and return their covariance matrix.
 
+    To construct a new kernel from the old ones, we can use Python operators
+    `+` or `*` with another kernel/constant. Or we can use methods `.add(...)`,
```

Comment:
You could use sphinx notation:
```
:meth:`add`, :meth:`mul`, ...
```

Reply:
Thanks, @fritzo , didn't know about that. Just discovered that we can type some math with :math:`...` too.